app/models/job_posting.rb

```diff
@@ -1,6 +1,8 @@
 class JobPosting < ApplicationRecord
   belongs_to :company
 
+  self.per_page = 10
```

Comment:
Where is this used? Also, if this is the number of opportunities per page we should consider increasing it to a multiple of the columns (if we have 3 columns then it should be either 9 or 12)

Reply:
It's used by will_paginate.